server/api/products.js

```diff
@@ -0,0 +1,24 @@
+const router = require("express").Router();
+const {
+  models: { Product },
+} = require("../db/index");
+
+router.get("/", async (req, res, next) => {
```

Comment:
I already wrote this same route, so this may be a conflict

Reply:
I will delete this on my side, and it won't conflict with yours.